optimum/bettertransformer/models/base.py

```diff
@@ -29,7 +32,21 @@
 
 
 class BetterTransformerBaseLayer(nn.Module):
-    def __init__(self, config):
+    def __init__(
+        self,
+        config: "transformers.PretrainedConfig",
```

Comment:
Can you add `from typing import TYPE_CHECKING` block?

Reply:
fixed in [34363bf](https://github.com/huggingface/optimum/pull/676/commits/34363bf8f9ad9c29ea0cdfae4580062afebf907d)